A build problem with qt-5.5

````
g++ -c -pipe -O2 -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong
--param=ssp-buffer-size=4 -g -O2 -pipe -W -Wall -Wchar-subscripts
-Wnon-virtual-dtor -Wno-long-long -Wcast-align -Wpointer-arith
-Wwrite-strings -Wformat-security -DNDEBUG -fvisibility=hidden
-fvisibility-inlines-hidden -std=c++0x -Wall -W -D_REENTRANT
 -fPIC -DQT_NO_DEBUG -DQT_PLUGIN -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB
  -I. -I/ROOT/qt5/immodule -I/ROOT -I/ROOT/uim -I/ROOT -I/ROOT/uim
  -I/ROOT/replace -I../../qt4/candwin -I../../qt4/immodule
  -isystem /usr/include/qt -isystem /usr/include/qt/QtWidgets -isystem
  /usr/include/qt/QtGui/5.5.0 -isystem /usr/include/qt/QtGui/5.5.0/QtGui
   -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtCore
   -I/ROOT/qt5/immodule -I/usr/lib/qt/mkspecs/linux-g++ -o plugin.o ../../qt4/immodule/plugin.cpp

In file included from /usr/include/qt/QtGui/qregion.h:42:0,
                 from /usr/include/qt/QtGui/qevent.h:39,
                 from /usr/include/qt/QtGui/QInputMethodEvent:1,
                 from ./quimplatforminputcontext.h:37,
                 from ../../qt4/immodule/quiminputcontext_with_slave.h:41,
                 from ../../qt4/immodule/plugin.cpp:66:
/usr/include/qt/QtCore/qdatastream.h:42:2: error: #error qdatastream.h must be included before any header file that defines Status
 #error qdatastream.h must be included before any header file that defines Status
  ^
In file included from /ROOT/uim/uim-x-util.h:37:0,
                 from ../../qt4/immodule/plugin.cpp:51:
/usr/include/qt/QtCore/qdatastream.h:97:10: error: expected identifier before ‘int’
     enum Status {
          ^
In file included from /usr/include/qt/QtGui/qregion.h:42:0,
                 from /usr/include/qt/QtGui/qevent.h:39,
                 from /usr/include/qt/QtGui/QInputMethodEvent:1,
                 from ./quimplatforminputcontext.h:37,
                 from ../../qt4/immodule/quiminputcontext_with_slave.h:41,
                 from ../../qt4/immodule/plugin.cpp:66:
/usr/include/qt/QtCore/qdatastream.h:97:17: error: expected unqualified-id before ‘{’ token
     enum Status {
                 ^
In file included from /ROOT/uim/uim-x-util.h:37:0,
                 from ../../qt4/immodule/plugin.cpp:51:
/usr/include/qt/QtCore/qdatastream.h: In function ‘QDataStream& operator>>(QDataStream&, QHash<T, U>&)’:
/usr/include/qt/QtCore/qdatastream.h:332:18: error: expected unqualified-id before ‘int’
     QDataStream::Status oldStatus = in.status();
                  ^
In file included from /usr/include/qt/QtGui/qregion.h:42:0,
                 from /usr/include/qt/QtGui/qevent.h:39,
                 from /usr/include/qt/QtGui/QInputMethodEvent:1,
                 from ./quimplatforminputcontext.h:37,
                 from ../../qt4/immodule/quiminputcontext_with_slave.h:41,
                 from ../../qt4/immodule/plugin.cpp:66:
/usr/include/qt/QtCore/qdatastream.h:340:28: error: ‘Ok’ is not a member of ‘QDataStream’
         if (in.status() != QDataStream::Ok)
                            ^
/usr/include/qt/QtCore/qdatastream.h:349:24: error: ‘Ok’ is not a member of ‘QDataStream’
     if (in.status() != QDataStream::Ok)
                        ^
/usr/include/qt/QtCore/qdatastream.h:351:9: error: ‘oldStatus’ was not declared in this scope
     if (oldStatus != QDataStream::Ok)
         ^
/usr/include/qt/QtCore/qdatastream.h:351:22: error: ‘Ok’ is not a member of ‘QDataStream’
     if (oldStatus != QDataStream::Ok)
                      ^
In file included from /ROOT/uim/uim-x-util.h:37:0,
                 from ../../qt4/immodule/plugin.cpp:51:
/usr/include/qt/QtCore/qdatastream.h: In function ‘QDataStream& operator>>(QDataStream&, QMap<T, U>&)’:
/usr/include/qt/QtCore/qdatastream.h:376:18: error: expected unqualified-id before ‘int’
     QDataStream::Status oldStatus = in.status();
                  ^
In file included from /usr/include/qt/QtGui/qregion.h:42:0,
                 from /usr/include/qt/QtGui/qevent.h:39,
                 from /usr/include/qt/QtGui/QInputMethodEvent:1,
                 from ./quimplatforminputcontext.h:37,
                 from ../../qt4/immodule/quiminputcontext_with_slave.h:41,
                 from ../../qt4/immodule/plugin.cpp:66:
/usr/include/qt/QtCore/qdatastream.h:385:28: error: ‘Ok’ is not a member of ‘QDataStream’
         if (in.status() != QDataStream::Ok)
                            ^
/usr/include/qt/QtCore/qdatastream.h:393:24: error: ‘Ok’ is not a member of ‘QDataStream’
     if (in.status() != QDataStream::Ok)
                        ^
/usr/include/qt/QtCore/qdatastream.h:395:9: error: ‘oldStatus’ was not declared in this scope
     if (oldStatus != QDataStream::Ok)
         ^
/usr/include/qt/QtCore/qdatastream.h:395:22: error: ‘Ok’ is not a member of ‘QDataStream’
     if (oldStatus != QDataStream::Ok)
                      ^
../../qt4/immodule/plugin.cpp: In member function ‘void UimInputContextPlugin::uimInit()’:
../../qt4/immodule/plugin.cpp:148:45: error: ‘QX11Info’ has not been declared
      uim_scm_callf("%xkb-set-display", "p", QX11Info::display());
                                             ^
Makefile.qmake:560: recipe for target 'plugin.o' failed
make[3]: *** [plugin.o] Error 1
make[3]: Leaving directory '/ROOT/qt5/immodule'
Makefile:610: recipe for target 'all' failed
make[2]: *** [all] Error 2
make[2]: Leaving directory '/ROOT/qt5/immodule'
Makefile:520: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/ROOT/qt5'
Makefile:652: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1

````

--- a/qt4/immodule/plugin.cpp
+++ b/qt4/immodule/plugin.cpp
@@ -46,6 +46,7 @@
 # if defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
 #  include <QtX11Extras/QX11Info>
 # endif
+# include <QtCore/qdatastream.h>
 # include <qpa/qplatforminputcontext.h>
 #endif
 

